src/scmdata/ops.py

```diff
@@ -597,6 +599,78 @@ def integrate(self, out_var=None):
     return out
 
 
+def delta_per_delta_time(self, out_var=None):
+    """
+    Calculate change in timeseries values for each timestep, divided by the size of the timestep
+
+    The output is placed on the middle of each timestep and is one timestep
```

Comment:
Is this normal? I assumed that a first order finite difference uses the same timesteps (except dropping the first/last).

Reply:
I have no idea. I couldn't really see a normal example i.e. where someone else had done this with timeseries. Pandas convention seems to be that you can specify how you want the diff to work https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.diff.html. In my head, putting it on the middle of the time axis seemed to make most sense. For example, if we had co2mass on 1st Jan, then we want to calculate flux into the atmosphere based on that, if it comes back on the middle of the year that makes it easy to then add with other fluxes which are middle of year. We could add extra options to specify how to calculate the new times, or we could just leave it to users to tweak times once the operation has been done (also easy).

Reply:
I agree that it does make more sense moving the timestep, but I just haven't seen it done before. For annual data you could just use run["year"] which won't change

Reply:
True. I actually don't have a use case for this yet so maybe we'll tweak soon anyway.